packages/app-desktop/gui/styles/index.scss

```diff
@@ -0,0 +1,2 @@
+
+@use './dialog-modal-layer.scss';
```

Comment:
RSCSS specifies that [one top-level rule should be in each file](https://ricostacruz.com/rscss/css-structure.html). I've created a new `styles/` folder to satisfy this rule. Note, however, that a **style/** folder already exists, and contains styled component logic. As such, it might make sense to move `styles/dialog-modal-layer.scss` to a different location.

Additionally, RSCSS states that [glob matching should be used](https://ricostacruz.com/rscss/css-structure.html). However, Dart SCSS does not seem to support globs in `@use`.